Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -326,21 +327,78 @@ PhysicalParticleContainer::AddPlasmaFromFile(const std::string s_f, amrex::Real
                                      "should contain only one species\n");
     std::pair<std::string,openPMD::ParticleSpecies> ps = *i.particles.begin();
 
-    //TODO: In future PRs will add AMREX_ALWAYS_ASSERT_WITH_MESSAGE to test if mass and charge are both const
+    //TODO: In future PRs will add ASSERT_WITH_MESSAGE to test if mass and charge are both const
     amrex::Real p_m = ps.second["mass"][openPMD::RecordComponent::SCALAR].loadChunk<amrex::Real>().get()[0];
     amrex::Real p_q = ps.second["charge"][openPMD::RecordComponent::SCALAR].loadChunk<amrex::Real>().get()[0];
-    int npart = ps.second["position"]["x"].getExtent()[0];
+    auto npart = ps.second["position"]["x"].getExtent()[0];
```

Comment:
sorry I overlooked this: actually even `auto const npart = ...;`

Reply:
This is fixed in this commit!